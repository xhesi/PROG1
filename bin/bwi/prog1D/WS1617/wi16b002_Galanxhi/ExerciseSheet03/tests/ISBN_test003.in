9780321751040
