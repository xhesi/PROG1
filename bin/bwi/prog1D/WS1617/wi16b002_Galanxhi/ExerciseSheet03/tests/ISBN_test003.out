enter ISBN: valid pattern: true
valid checkDigit: false
